src/main/java/net/dv8tion/jda/core/MessageBuilder.java

```diff
@@ -366,18 +431,505 @@ public int getLength()
+     * @return this instance
+     */
+    public MessageBuilder replaceLast(String target, String replacement)
+    {
+        int index = builder.lastIndexOf(target);
+        if (index != -1)
+        {
+            builder.replace(index, index + target.length(), replacement);
+        }
+        return this;
+    }
+
+    /**
+     * Removes all mentions and replaces them with the closest looking textual representation.<p>
+     * <p>
+     * Use this over {@link #stripMentions(Guild)} if {@link MentionType.USER User} mentions should be replaced with their user names
+     *
+     * @param jda the JDA instance
+     * @return this instance
+     */
+    public MessageBuilder stripMentions(JDA jda) {
+        return this.stripMentions(jda, (Guild) null, MentionType.EVERYONE, MentionType.HERE, MentionType.CHANNEL, MentionType.ROLE, MentionType.USER);
+    }
+
+    /**
+     * Removes all mentions and replaces them with the closest looking textual representation.<p>
+     * <p>
+     * Use this over {@link #stripMentions(JDA)} if {@link MentionType.USER User} mentions should be replaced with their nicknames in a specific guild
+     *
+     * @param guild the guild for {@link MentionType.USER User} mentions 
+     * @return this instance
+     */
+    public MessageBuilder stripMentions(Guild guild) {
+        return this.stripMentions(guild.getJDA(), guild, MentionType.EVERYONE, MentionType.HERE, MentionType.CHANNEL, MentionType.ROLE, MentionType.USER);
+    }
+
+    /**
+     * Removes all mentions of the specified types and replaces them with the closest looking textual representation.<p>
+     * <p>
+     * Use this over {@link #stripMentions(JDA, MentionType...)} if {@link MentionType.USER User} mentions should be replaced with their nicknames in a specific guild
+     * 
+     * @param guild the guild for {@link MentionType.USER User} mentions 
+     * @return this instance
+     */
+    public MessageBuilder stripMentions(Guild guild, MentionType... types) {
+        return this.stripMentions(guild.getJDA(), guild, types);
+    }
+
+    /**
+     * Removes all mentions of the specified types and replaces them with the closest looking textual representation.<p>
+     * <p>
+     * Use this over {@link #stripMentions(Guild, MentionType...)} if {@link MentionType.USER User} mentions should be replaced with their user names
+     * 
+     * @param jda the {@link net.dv8tion.jda.core.JDA JDA} instance, only needed for {@link MentionType.USER User}, {@link MentionType.CHANNEL Channel} and {@link MentionType.GUILD Guild} mentions
+     * @param types the {@link MentionType MentionTypes} that should be stripped
+     * @return this instance
+     */
+    public MessageBuilder stripMentions(JDA jda, MentionType... types) {
+        return this.stripMentions(jda, (Guild) null, types);
+    }
+
+    private MessageBuilder stripMentions(JDA jda, Guild guild, MentionType... types) {
+        String string = null;
+        if (types != null)
```

Comment:
if yo just call `#stripMentions(jda, guild, null)` the `types` argument will be null, not an empty array